### DISCUSSION TITLE
使用带验证的表单时报错

### DISCUSSION THREAD

**Discussion by MinCpopo** [TaleLin/lin-ui#1201] [求助] [answered]
点击输入框后会报错：TypeError: Cannot read property 'selectComponent' of undefined
![image](https://user-images.githubusercontent.com/42403875/102873555-40e03100-447c-11eb-9ccc-dfa23489ccf0.png)
![image](https://user-images.githubusercontent.com/42403875/102873602-4b022f80-447c-11eb-9436-b17ba610d7f0.png)

**Comment by murongg** [comment, marked as answer]
参考此issue：https://github.com/TaleLin/lin-ui/issues/1050
